cva6/sim/Makefile

```diff
@@ -25,11 +25,24 @@ FLIST_TB   := $(CVA6_TB_DIR)/Flist.cva6_tb
 target     ?= cv64a6_imafdc_sv39
 FLIST_CORE := $(CVA6_REPO_DIR)/core/Flist.$(target)
 
-VERDI          ?=
-user-define    ?= +define+WT_CACHE
-USERDEFINE      = +define+RVFI_TRACE $(user-define)
+VERDI           ?=
+path-var        ?=
+tool_path       ?=
+isscomp_opts    ?=
+issrun_opts     ?=
+isspostrun_opts ?=
+log             ?=
+variant         ?=
+
+TESTNAME := $(shell basename -s .o $(elf))
+
+ifeq ($(isspostrun_opts), "")
```

Comment:
If 'isspostrun_opts' is empty, the invocation of 'grep' will run forever.  Change the condition from 'ifeq' to 'ifneq' (this way the code will read more naturally than if the true and false branches were swapped.)